heron/spi/src/java/BUILD

```diff
@@ -146,3 +146,19 @@ java_library(
     ),
     deps = statemgr_deps_files,
 )
+
+java_binary(
+    name = "spi-unshaded",
+    srcs = glob([
+        "**/spi/utils/**/*.java",
+        "**/spi/common/**/*.java",
+    ]),
+    deps = utils_deps_files,
+)
+
+genrule(
+    name = "heron-spi-jar",
+    srcs = [":spi-unshaded_deploy.jar"],
+    outs = ["heron-spi.jar"],
+    cmd  = "cp $< $@",
+)
```

Comment:
Ah sorry I misunderstood. I can look into adding it there as well.

On Friday, June 17, 2016, Karthik Ramasamy <notifications@github.com> wrote:

> In heron/spi/src/java/BUILD
> <https://github.com/twitter/heron/pull/937#discussion_r67466587>:
>
> > +
> > +java_binary(
> > +    name = "spi-unshaded",
> > +    srcs = glob([
> > +        "**/spi/utils/**/*.java",
> > +        "**/spi/common/**/*.java",
> > +    ]),
> > +    deps = utils_deps_files,
> > +)
> > +
> > +genrule(
> > +    name = "heron-spi-jar",
> > +    srcs = [":spi-unshaded_deploy.jar"],
> > +    outs = ["heron-spi.jar"],
> > +    cmd  = "cp $< $@",
> > +)
>
> We need to get this jar into heron-api install scripts - so that when a
> user installs he/she gets everything.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/twitter/heron/pull/937/files/b4087fad325028efed0c3a317bd786894a4f1bd2#r67466587>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfHUyQ3gTVlLHq9HCXpDGZTr94GEPD_ks5qMkNXgaJpZM4I3zHj>
> .
>


-- 
----
Andrew Jorgensen
@ajorgensen